.golangci.yml

```diff
@@ -5,11 +5,21 @@ linters-settings:
   misspell:
     ignore-words:
       - ect
+  depguard:
+    rules:
+      quicvarint:
+        list-mode: strict
+        files:
+          - "**/github.com/quic-go/quic-go/quicvarint/*"
+          - "!$test"
+        allow:
+          - $gostd
```

Comment:
Adding a validation here. This package shouldn't need to use any packages outside of the standard library.